Namco/map.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
```

Comment:
Ezért is szokott sírni a pycharm, hogy Classok elé/után 2 üres sor kell. De amúgy ha nyomsz egy Ctrl+Alt+L-t, akkor ezeket megcsinálja magától a pycharm

Reply:
Köszi, javítva.